client/src/components/ProtectedRoute/AuthProtectedRoute.tsx

```diff
@@ -9,7 +9,9 @@ interface AuthProtectedRouteProps extends RouteProps {
 const AuthProtectedRoute: React.FC<AuthProtectedRouteProps> = ({ ...rest }) => {
   const { loggedInUser } = useAuth();
 
-  if (loggedInUser !== null && loggedInUser !== undefined) return <Redirect to="/dashboard" />;
+  if (loggedInUser) return <Redirect to="/dashboard" />;
```

Comment:
Sorry, this should be `if (!loggedInUser)`. I forgot the bang operator there. In former code line,  `if (loggedInUser !== null && loggedInUser !== undefined)`, it's not possible to have one variable `loggedInUser` have the values `null` and `undefined` at the same time. So I forgot to mention that. Just update the code with `if (!loggedInUser)` :smile:

Reply:
Hi @ezeilo-su , I tried this and I can only see this now. 
![circular](https://user-images.githubusercontent.com/47619833/142654126-de2bf3ba-b49f-469e-8da1-1b4c52a917c2.PNG)

Reply:
@ezeilo-su @gagandeep95 If a user is not logged in , we want to redirect them to the login page (not the dashboard). On the other hand, if they are logged in we want to render the appropriate route